### PR TITLE
Use isinstance() rather than type() for a typecheck.

### PR DESCRIPTION
**Pull request by MaxJPRey**

### CODE REVIEW

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, could you update the code to leverage the fact that `isinstance` can work with a tuple of classes for the second ?  It'll return true if the first argument is an instance of any of the types in that sequence.

**Comment by MaxJPRey** [comment]
> LGTM, could you update the code to leverage the fact that `isinstance` can work with a tuple of classes for the second ? It'll return true if the first argument is an instance of any of the types in that sequence.

Excellent idea @SMoraisAnsys . I sort of automated the process.

File: pyaedt/modeler/cad/Primitives.py [hunk 1]
```diff
@@ -2451,7 +2451,7 @@ def get_group_bounding_box(self, group):
         >>> oEditor.GetObjectsInGroup
         >>> oEditor.GetModelBoundingBox
         """
-        if type(group) is not str:
+        if isinstance(group, str):
```
Comment (MaxJPRey) [inline pyaedt/modeler/cad/Primitives.py:2454]:
```suggestion
        if not isinstance(group, str):
```

File: pyaedt/modules/Mesh.py [hunk 2]
```diff
@@ -244,10 +244,11 @@ def update_assignment(self):
         if "Faces" in self.props:
             faces = self.props["Faces"]
             faces_out = []
-            if type(faces) is not list:
+            if not isinstance(faces, list):
                 faces = [faces]
             for f in faces:
-                if type(f) is EdgePrimitive or type(f) is FacePrimitive or type(f) is VertexPrimitive:
+
```
Comment (MaxJPRey) [inline pyaedt/modules/Mesh.py:250]:
```suggestion
```

**Review by SMoraisAnsys — DISMISSED** [review]

**Review by myoung301 — DISMISSED** [review]
The EMIT-related portions look good to me.

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

**Pull request merged**